study.md

```diff
@@ -134,15 +134,16 @@ material. Instead, digest what you've read and repeat it in your own voice.
 In your own words, give a brief description of what HTTP is.
 
 ```md
-<!-- your answer here -->
+HTTP is how webpages are sent and received.
```

Comment:
Correct, more specifically it is the set of rules that determine how requests and responses are sent.